### ISSUE TITLE
requests.exceptions.ConnectionError - when importing data

### ISSUE THREAD

**Issue by bkocis** [weaviate/Verba#9]
When executing the import data command I get an connection error, but when I run the same command again, it passes. However, it appears that the application does read in the information from the data.txt. 

I don't set `VERBA_URL`, `VERBA_API_KEY`

Command that I execute: 
`verba import --path "/home/snow/Documents/Projects/weaviate-verba/resources/" --model "gpt-3.5-turbo" --clear True`

Traceback: 

```bash
verba import --path "/home/snow/Documents/Projects/weaviate-verba/resources/" --model "gpt-3.5-turbo" --clear True

===================== Creating Document and Chunk class =====================
ℹ Setting up client
ℹ VERBA_URL environment variable not set. Using Weaviate Embedded
Started ./.verba/cache/weaviate-embedded: process ID 200093
{"action":"startup","default_vectorizer_module":"none","level":"info","msg":"the default vectorizer modules is set to \"none\", as a result all new schema classes without an explicit vectorizer setting, will use this vectorizer","time":"2023-09-08T00:11:07+02:00"}
{"action":"startup","auto_schema_enabled":true,"level":"info","msg":"auto schema enabled setting is set to \"true\"","time":"2023-09-08T00:11:07+02:00"}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"cache_d8Z8DfRI0d6H","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:11:07+02:00","took":140330}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"chunk_ZFmXy4onugDM","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:11:07+02:00","took":87042}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"document_0dHcUXJompMJ","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:11:07+02:00","took":90100}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"suggestion_aJSePokVcdxW","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:11:07+02:00","took":104632}
{"level":"warning","msg":"Multiple vector spaces are present, GraphQL Explore and REST API list objects endpoint module include params has been disabled as a result.","time":"2023-09-08T00:11:07+02:00"}
{"action":"grpc_startup","level":"info","msg":"grpc server listening at [::]:50051","time":"2023-09-08T00:11:07+02:00"}
{"action":"restapi_management","level":"info","msg":"Serving weaviate at http://127.0.0.1:6666","time":"2023-09-08T00:11:07+02:00"}
✔ Client connected to local Weaviate server
Document class already exists, do you want to overwrite it? (y/n): y
{"action":"hnsw_vector_cache_prefill","count":1000,"index_id":"document_at2uxArUgLG1","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:11:11+02:00","took":182906}
{"action":"hnsw_vector_cache_prefill","count":1000,"index_id":"chunk_L4xfAO6z3nSu","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:11:11+02:00","took":170259}
✔ 'Document' and 'Chunk' schemas created
{"action":"restapi_management","level":"info","msg":"Shutting down... ","time":"2023-09-08T00:11:11+02:00"}
{"action":"restapi_management","level":"info","msg":"Stopped serving weaviate at http://127.0.0.1:6666","time":"2023-09-08T00:11:11+02:00"}
ℹ Done

============================ Creating Cache class ============================
ℹ Setting up client
ℹ VERBA_URL environment variable not set. Using Weaviate Embedded
Started ./.verba/cache/weaviate-embedded: process ID 200235
{"action":"startup","default_vectorizer_module":"none","level":"info","msg":"the default vectorizer modules is set to \"none\", as a result all new schema classes without an explicit vectorizer setting, will use this vectorizer","time":"2023-09-08T00:11:11+02:00"}
{"action":"startup","auto_schema_enabled":true,"level":"info","msg":"auto schema enabled setting is set to \"true\"","time":"2023-09-08T00:11:11+02:00"}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"cache_d8Z8DfRI0d6H","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:11:11+02:00","took":59862}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"chunk_L4xfAO6z3nSu","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:11:11+02:00","took":38679}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"document_at2uxArUgLG1","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:11:11+02:00","took":38062}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"suggestion_aJSePokVcdxW","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:11:11+02:00","took":35334}
{"level":"warning","msg":"Multiple vector spaces are present, GraphQL Explore and REST API list objects endpoint module include params has been disabled as a result.","time":"2023-09-08T00:11:11+02:00"}
{"action":"grpc_startup","level":"info","msg":"grpc server listening at [::]:50051","time":"2023-09-08T00:11:11+02:00"}
{"action":"restapi_management","level":"info","msg":"Serving weaviate at http://127.0.0.1:6666","time":"2023-09-08T00:11:11+02:00"}
✔ Client connected to local Weaviate server
Cache class already exists, do you want to overwrite it? (y/n): y
{"action":"hnsw_vector_cache_prefill","count":1000,"index_id":"cache_n3X4UDflnGSR","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:11:13+02:00","took":108556}
✔ 'Cache' schema created
{"action":"restapi_management","level":"info","msg":"Shutting down... ","time":"2023-09-08T00:11:13+02:00"}
{"action":"restapi_management","level":"info","msg":"Stopped serving weaviate at http://127.0.0.1:6666","time":"2023-09-08T00:11:13+02:00"}
ℹ Done

========================= Creating Suggestion class =========================
ℹ Setting up client
ℹ VERBA_URL environment variable not set. Using Weaviate Embedded
Started ./.verba/cache/weaviate-embedded: process ID 200307
{"action":"startup","default_vectorizer_module":"none","level":"info","msg":"the default vectorizer modules is set to \"none\", as a result all new schema classes without an explicit vectorizer setting, will use this vectorizer","time":"2023-09-08T00:11:13+02:00"}
{"action":"startup","auto_schema_enabled":true,"level":"info","msg":"auto schema enabled setting is set to \"true\"","time":"2023-09-08T00:11:13+02:00"}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"cache_n3X4UDflnGSR","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:11:13+02:00","took":56962}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"chunk_L4xfAO6z3nSu","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:11:13+02:00","took":55732}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"document_at2uxArUgLG1","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:11:13+02:00","took":84016}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"suggestion_aJSePokVcdxW","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:11:13+02:00","took":60599}
{"level":"warning","msg":"Multiple vector spaces are present, GraphQL Explore and REST API list objects endpoint module include params has been disabled as a result.","time":"2023-09-08T00:11:13+02:00"}
{"action":"grpc_startup","level":"info","msg":"grpc server listening at [::]:50051","time":"2023-09-08T00:11:13+02:00"}
{"action":"restapi_management","level":"info","msg":"Serving weaviate at http://127.0.0.1:6666","time":"2023-09-08T00:11:13+02:00"}
✔ Client connected to local Weaviate server
Suggestion class already exists, do you want to overwrite it? (y/n): y
{"action":"hnsw_vector_cache_prefill","count":1000,"index_id":"suggestion_JvWuPHTJUNPN","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:11:14+02:00","took":199752}
✔ 'Suggestion' schema created
{"action":"restapi_management","level":"info","msg":"Shutting down... ","time":"2023-09-08T00:11:14+02:00"}
{"action":"restapi_management","level":"info","msg":"Stopped serving weaviate at http://127.0.0.1:6666","time":"2023-09-08T00:11:14+02:00"}
ℹ Done

============================ Starting data import ============================
ℹ Setting up client
ℹ VERBA_URL environment variable not set. Using Weaviate Embedded
Started ./.verba/cache/weaviate-embedded: process ID 200378
{"action":"startup","default_vectorizer_module":"none","level":"info","msg":"the default vectorizer modules is set to \"none\", as a result all new schema classes without an explicit vectorizer setting, will use this vectorizer","time":"2023-09-08T00:11:14+02:00"}
{"action":"startup","auto_schema_enabled":true,"level":"info","msg":"auto schema enabled setting is set to \"true\"","time":"2023-09-08T00:11:14+02:00"}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"cache_n3X4UDflnGSR","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:11:14+02:00","took":39691}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"chunk_L4xfAO6z3nSu","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:11:14+02:00","took":54810}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"document_at2uxArUgLG1","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:11:14+02:00","took":34729}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"suggestion_JvWuPHTJUNPN","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:11:14+02:00","took":48080}
{"level":"warning","msg":"Multiple vector spaces are present, GraphQL Explore and REST API list objects endpoint module include params has been disabled as a result.","time":"2023-09-08T00:11:14+02:00"}
{"action":"grpc_startup","level":"info","msg":"grpc server listening at [::]:50051","time":"2023-09-08T00:11:14+02:00"}
{"action":"restapi_management","level":"info","msg":"Serving weaviate at http://127.0.0.1:6666","time":"2023-09-08T00:11:14+02:00"}
✔ Client connected to local Weaviate server
ℹ All schemas available
ℹ Reading
/home/snow/Documents/Projects/weaviate-verba/resources/data.txt
✔ Loaded 1 files
ℹ Converted
/home/snow/Documents/Projects/weaviate-verba/resources/data.txt
✔ All 1 files successfully loaded
ℹ Starting splitting process
✔ Successful splitting (total 0)
ℹ (1/1) Importing document
/home/snow/Documents/Projects/weaviate-verba/resources/data.txt
✔ Imported all docs
✔ Imported all chunks
{"action":"restapi_management","level":"info","msg":"Shutting down... ","time":"2023-09-08T00:11:14+02:00"}
{"action":"restapi_management","level":"info","msg":"Stopped serving weaviate at http://127.0.0.1:6666","time":"2023-09-08T00:11:14+02:00"}
Exception in thread batchSizeRefresh:
Traceback (most recent call last):
  File "/home/snow/venv/weaviate-verba/lib/python3.10/site-packages/urllib3/connection.py", line 203, in _new_conn
    sock = connection.create_connection(
  File "/home/snow/venv/weaviate-verba/lib/python3.10/site-packages/urllib3/util/connection.py", line 85, in create_connection
    raise err
  File "/home/snow/venv/weaviate-verba/lib/python3.10/site-packages/urllib3/util/connection.py", line 73, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/snow/venv/weaviate-verba/lib/python3.10/site-packages/urllib3/connectionpool.py", line 790, in urlopen
    response = self._make_request(
  File "/home/snow/venv/weaviate-verba/lib/python3.10/site-packages/urllib3/connectionpool.py", line 496, in _make_request
    conn.request(
  File "/home/snow/venv/weaviate-verba/lib/python3.10/site-packages/urllib3/connection.py", line 395, in request
    self.endheaders()
  File "/usr/lib/python3.10/http/client.py", line 1278, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.10/http/client.py", line 1038, in _send_output
    self.send(msg)
  File "/usr/lib/python3.10/http/client.py", line 976, in send
    self.connect()
  File "/home/snow/venv/weaviate-verba/lib/python3.10/site-packages/urllib3/connection.py", line 243, in connect
    self.sock = self._new_conn()
  File "/home/snow/venv/weaviate-verba/lib/python3.10/site-packages/urllib3/connection.py", line 218, in _new_conn
    raise NewConnectionError(
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x7f5eadaec370>: Failed to establish a new connection: [Errno 111] Connection refused

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/snow/venv/weaviate-verba/lib/python3.10/site-packages/requests/adapters.py", line 486, in send
    resp = conn.urlopen(
  File "/home/snow/venv/weaviate-verba/lib/python3.10/site-packages/urllib3/connectionpool.py", line 844, in urlopen
    retries = retries.increment(
  File "/home/snow/venv/weaviate-verba/lib/python3.10/site-packages/urllib3/util/retry.py", line 515, in increment
    raise MaxRetryError(_pool, url, reason) from reason  # type: ignore[arg-type]
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=6666): Max retries exceeded with url: /v1/nodes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5eadaec370>: Failed to establish a new connection: [Errno 111] Connection refused'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/snow/venv/weaviate-verba/lib/python3.10/site-packages/weaviate/cluster/cluster.py", line 62, in get_nodes_status
    response = self._connection.get(path=path)
  File "/home/snow/venv/weaviate-verba/lib/python3.10/site-packages/weaviate/connect/connection.py", line 539, in get
    return self._session.get(
  File "/home/snow/venv/weaviate-verba/lib/python3.10/site-packages/requests/sessions.py", line 602, in get
    return self.request("GET", url, **kwargs)
  File "/home/snow/venv/weaviate-verba/lib/python3.10/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/snow/venv/weaviate-verba/lib/python3.10/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/home/snow/venv/weaviate-verba/lib/python3.10/site-packages/requests/adapters.py", line 519, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='localhost', port=6666): Max retries exceeded with url: /v1/nodes (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5eadaec370>: Failed to establish a new connection: [Errno 111] Connection refused'))

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/snow/venv/weaviate-verba/lib/python3.10/site-packages/weaviate/batch/crud_batch.py", line 442, in periodic_check
    status = cluster.get_nodes_status()
  File "/home/snow/venv/weaviate-verba/lib/python3.10/site-packages/weaviate/cluster/cluster.py", line 64, in get_nodes_status
    raise RequestsConnectionError(
requests.exceptions.ConnectionError: Get nodes status failed due to connection error

```

after simply repeating the command: 

```bash 
===================== Creating Document and Chunk class =====================
ℹ Setting up client
ℹ VERBA_URL environment variable not set. Using Weaviate Embedded
Started ./.verba/cache/weaviate-embedded: process ID 205223
{"action":"startup","default_vectorizer_module":"none","level":"info","msg":"the default vectorizer modules is set to \"none\", as a result all new schema classes without an explicit vectorizer setting, will use this vectorizer","time":"2023-09-08T00:14:46+02:00"}
{"action":"startup","auto_schema_enabled":true,"level":"info","msg":"auto schema enabled setting is set to \"true\"","time":"2023-09-08T00:14:46+02:00"}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"cache_n3X4UDflnGSR","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:14:46+02:00","took":192734}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"chunk_L4xfAO6z3nSu","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:14:46+02:00","took":180536}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"document_at2uxArUgLG1","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:14:46+02:00","took":261801}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"suggestion_JvWuPHTJUNPN","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:14:46+02:00","took":173080}
{"level":"warning","msg":"Multiple vector spaces are present, GraphQL Explore and REST API list objects endpoint module include params has been disabled as a result.","time":"2023-09-08T00:14:46+02:00"}
{"action":"grpc_startup","level":"info","msg":"grpc server listening at [::]:50051","time":"2023-09-08T00:14:46+02:00"}
{"action":"restapi_management","level":"info","msg":"Serving weaviate at http://127.0.0.1:6666","time":"2023-09-08T00:14:46+02:00"}
✔ Client connected to local Weaviate server
Document class already exists, do you want to overwrite it? (y/n): y
{"action":"hnsw_vector_cache_prefill","count":1000,"index_id":"document_4KF2RtSb7AzF","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:14:49+02:00","took":50649}
{"action":"hnsw_vector_cache_prefill","count":1000,"index_id":"chunk_afMWBLITi7Hg","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:14:49+02:00","took":36439}
✔ 'Document' and 'Chunk' schemas created
{"action":"restapi_management","level":"info","msg":"Shutting down... ","time":"2023-09-08T00:14:49+02:00"}
{"action":"restapi_management","level":"info","msg":"Stopped serving weaviate at http://127.0.0.1:6666","time":"2023-09-08T00:14:49+02:00"}
ℹ Done

============================ Creating Cache class ============================
ℹ Setting up client
ℹ VERBA_URL environment variable not set. Using Weaviate Embedded
Started ./.verba/cache/weaviate-embedded: process ID 205337
{"action":"startup","default_vectorizer_module":"none","level":"info","msg":"the default vectorizer modules is set to \"none\", as a result all new schema classes without an explicit vectorizer setting, will use this vectorizer","time":"2023-09-08T00:14:49+02:00"}
{"action":"startup","auto_schema_enabled":true,"level":"info","msg":"auto schema enabled setting is set to \"true\"","time":"2023-09-08T00:14:49+02:00"}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"cache_n3X4UDflnGSR","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:14:49+02:00","took":53770}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"chunk_afMWBLITi7Hg","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:14:49+02:00","took":49957}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"document_4KF2RtSb7AzF","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:14:49+02:00","took":48970}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"suggestion_JvWuPHTJUNPN","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:14:49+02:00","took":51621}
{"level":"warning","msg":"Multiple vector spaces are present, GraphQL Explore and REST API list objects endpoint module include params has been disabled as a result.","time":"2023-09-08T00:14:49+02:00"}
{"action":"grpc_startup","level":"info","msg":"grpc server listening at [::]:50051","time":"2023-09-08T00:14:49+02:00"}
{"action":"restapi_management","level":"info","msg":"Serving weaviate at http://127.0.0.1:6666","time":"2023-09-08T00:14:49+02:00"}
✔ Client connected to local Weaviate server
Cache class already exists, do you want to overwrite it? (y/n): y
{"action":"hnsw_vector_cache_prefill","count":1000,"index_id":"cache_dE7829wgUrT0","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:14:50+02:00","took":80944}
✔ 'Cache' schema created
{"action":"restapi_management","level":"info","msg":"Shutting down... ","time":"2023-09-08T00:14:50+02:00"}
{"action":"restapi_management","level":"info","msg":"Stopped serving weaviate at http://127.0.0.1:6666","time":"2023-09-08T00:14:50+02:00"}
ℹ Done

========================= Creating Suggestion class =========================
ℹ Setting up client
ℹ VERBA_URL environment variable not set. Using Weaviate Embedded
Started ./.verba/cache/weaviate-embedded: process ID 205405
{"action":"startup","default_vectorizer_module":"none","level":"info","msg":"the default vectorizer modules is set to \"none\", as a result all new schema classes without an explicit vectorizer setting, will use this vectorizer","time":"2023-09-08T00:14:50+02:00"}
{"action":"startup","auto_schema_enabled":true,"level":"info","msg":"auto schema enabled setting is set to \"true\"","time":"2023-09-08T00:14:50+02:00"}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"cache_dE7829wgUrT0","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:14:50+02:00","took":41406}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"chunk_afMWBLITi7Hg","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:14:50+02:00","took":49932}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"document_4KF2RtSb7AzF","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:14:50+02:00","took":47882}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"suggestion_JvWuPHTJUNPN","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:14:50+02:00","took":49368}
{"level":"warning","msg":"Multiple vector spaces are present, GraphQL Explore and REST API list objects endpoint module include params has been disabled as a result.","time":"2023-09-08T00:14:50+02:00"}
{"action":"grpc_startup","level":"info","msg":"grpc server listening at [::]:50051","time":"2023-09-08T00:14:50+02:00"}
{"action":"restapi_management","level":"info","msg":"Serving weaviate at http://127.0.0.1:6666","time":"2023-09-08T00:14:50+02:00"}
✔ Client connected to local Weaviate server
Suggestion class already exists, do you want to overwrite it? (y/n): y
{"action":"hnsw_vector_cache_prefill","count":1000,"index_id":"suggestion_nRDqcQr9MEAo","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:14:51+02:00","took":186550}
✔ 'Suggestion' schema created
{"action":"restapi_management","level":"info","msg":"Shutting down... ","time":"2023-09-08T00:14:51+02:00"}
{"action":"restapi_management","level":"info","msg":"Stopped serving weaviate at http://127.0.0.1:6666","time":"2023-09-08T00:14:51+02:00"}
ℹ Done

============================ Starting data import ============================
ℹ Setting up client
ℹ VERBA_URL environment variable not set. Using Weaviate Embedded
Started ./.verba/cache/weaviate-embedded: process ID 205474
{"action":"startup","default_vectorizer_module":"none","level":"info","msg":"the default vectorizer modules is set to \"none\", as a result all new schema classes without an explicit vectorizer setting, will use this vectorizer","time":"2023-09-08T00:14:51+02:00"}
{"action":"startup","auto_schema_enabled":true,"level":"info","msg":"auto schema enabled setting is set to \"true\"","time":"2023-09-08T00:14:51+02:00"}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"cache_dE7829wgUrT0","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:14:51+02:00","took":196427}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"chunk_afMWBLITi7Hg","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:14:51+02:00","took":272944}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"document_4KF2RtSb7AzF","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:14:51+02:00","took":226739}
{"action":"hnsw_vector_cache_prefill","count":3000,"index_id":"suggestion_nRDqcQr9MEAo","level":"info","limit":1000000000000,"msg":"prefilled vector cache","time":"2023-09-08T00:14:51+02:00","took":157232}
{"level":"warning","msg":"Multiple vector spaces are present, GraphQL Explore and REST API list objects endpoint module include params has been disabled as a result.","time":"2023-09-08T00:14:51+02:00"}
{"action":"grpc_startup","level":"info","msg":"grpc server listening at [::]:50051","time":"2023-09-08T00:14:51+02:00"}
{"action":"restapi_management","level":"info","msg":"Serving weaviate at http://127.0.0.1:6666","time":"2023-09-08T00:14:51+02:00"}
✔ Client connected to local Weaviate server
ℹ All schemas available
ℹ Reading
/home/snow/Documents/Projects/weaviate-verba/resources/data.txt
✔ Loaded 1 files
ℹ Converted
/home/snow/Documents/Projects/weaviate-verba/resources/data.txt
✔ All 1 files successfully loaded
ℹ Starting splitting process
✔ Successful splitting (total 0)
ℹ (1/1) Importing document
/home/snow/Documents/Projects/weaviate-verba/resources/data.txt
✔ Imported all docs
✔ Imported all chunks
{"action":"restapi_management","level":"info","msg":"Shutting down... ","time":"2023-09-08T00:14:52+02:00"}
{"action":"restapi_management","level":"info","msg":"Stopped serving weaviate at http://127.0.0.1:6666","time":"2023-09-08T00:14:52+02:00"}

```

**Comment by thomashacker** [comment]
Thanks a lot! We'll look into this

**Comment by thomashacker** [comment]
This issue should be fixed with the new 0.3.0 version 🚀  Thanks a lot!
